src/Core/src/Platform/Android/ViewGroupExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using AView = Android.Views.View;
+using AViewGroup = Android.Views.ViewGroup;
+
+namespace Microsoft.Maui
+{
+	public static class ViewGroupExtensions
+	{
+		public static IEnumerable<T> GetChildrenOfType<T>(this AViewGroup self) where T : AView
+		{
+			for (var i = 0; i < self.ChildCount; i++)
```

Comment:
Not sure which is better, but iOS seems to follow a "wide" search and this is "deep". Probably not too essential, but just a difference.